cli/src/core/config.ts

```diff
@@ -1,14 +1,33 @@
+import { readFileSync } from 'node:fs';
 import { readFile } from 'node:fs/promises';
+import os from 'node:os';
+import path from 'node:path';
+import yaml from 'js-yaml';
 
 const info = JSON.parse(
   await readFile(new URL('../../package.json', import.meta.url), {
     encoding: 'utf8',
   }),
 );
 
+export const configDir = path.join(os.homedir(), '.cosmo');
```

Comment:
I'm not sure if this is the right approach for all systems. I'd take inspiration from this package https://github.com/sindresorhus/env-paths

Reply:
Done